src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
Doesn't look like the method is  extended in the repo, so changed it to `using`  for safety. Not  essential, though.

https://docs.julialang.org/en/stable/manual/modules/
"The import keyword supports all the same syntax as using, but only operates on a single name at a time. It does not add modules to be searched the way using does. import also differs from using in that functions must be imported using import to be extended with new methods."